### DISCUSSION TITLE
PORT bzw. Portweiterleitung für Webserver

### DISCUSSION THREAD

**Discussion by commander-bln** [jomjol/AI-on-the-edge-device#2072] [Ideas] [answered]
Hallo
mit normalen Boardmitteln schaffe ich es nicht, das der EPS32 CAM von ausserhalb der Netzwerkes erreichbar ist.
Die Weiterleitung auf die IP mit Fritzbox funktioniert, aber es erscheint keine Webseite, weder auf  Standard 80 noch auf einem beliebigen Port z.B. 5062 oder 5069

vielleicht läßt sich das in einem späteren Update mit integrieren.

**Comment by caco3** [comment, marked as answer]
Was ist dein Wunsch?

Wenn DU in der Fritzbox eine Weiterleitung (Aussen: beliebiger Port, innen: Port 80 auf der IP des devices) machst, sollte es ohne Probleme funktionieren.

@jomjol oder geht das nicht wegen deinen Test-funktionen? Ich meine, da mal was gelesen zu haben.

Wir werden im Gerät selber kein umbiegen der Ports bereitstellen.

**Comment by commander-bln** [comment]
Hallo, danke für die Info

Habe es nochmal probiert, es geht!!, keine Ahnung was 

Ich beim letzten Versuch falsch gemacht habe.

 

Danke

 

Von: CaCO3 ***@***.*** 
Gesendet: Samstag, 25. Februar 2023 20:45
An: jomjol/AI-on-the-edge-device
Cc: commander-bln; Author
Betreff: Re: [jomjol/AI-on-the-edge-device] PORT bzw. Portweiterleitung für Webserver (Discussion #2072)

 

Was ist dein Wunsch?

Wenn DU in der Fritzbox eine Weiterleitung (Aussen: beliebiger Port, innen: Port 80 auf der IP des devices) machst, sollte es ohne Probleme funktionieren.

@jomjol <https://github.com/jomjol>  oder geht das nicht wegen deinen Test-funktionen? Ich meine, da mal was gelesen zu haben.

Wir werden im Gerät selber kein umbiegen der Ports bereitstellen.
